bundles/statistics/statsgrid2016/MyIndicatorsTab.js

```diff
@@ -0,0 +1,316 @@
+            var me = this;
+            var btn = Oskari.clazz.create('Oskari.userinterface.component.Button');
+            btn.setTitle(this.loc('userIndicators.buttonTitle'));
+            btn.insertTo(this.content);
+            btn.setHandler(function (event) {
+                event.stopPropagation();
+                var formFlyout = me.instance.getFlyoutManager().getFlyout('indicatorForm');
+                formFlyout.showForm(me.userDsId);
+            });
+            return btn;
+        },
+        /**
+         * @private @method _renderIndicatorsList
+         * Renders given indicators list. Removes previous listing.
+         *
+         * @param {Object[]} indicators object array as returned by backend service
+         */
+        _renderIndicatorsList: function (indicators) {
+            if (!indicators) {
+                indicators = [];
+            }
+            this.listContainer.empty();
+            this.indicatorData = indicators;
+            var model = this._getGridModel(indicators);
+            var grid = this._getGrid(model);
+            grid.renderTo(this.listContainer);
+        },
+
+        /**
+         * @private @method _refreshIndicatorsList
+         * Fetches indicators from backend and renders the response.
+         */
+        _refreshIndicatorsList: function () {
+            var me = this;
+            me.service.getIndicatorList(me.userDsId, function (err, response) {
+                if (err) {
+                    me.log.warn('Could not list own indicators in personal data tab');
+                } else if (response && response.complete) {
+                    me._renderIndicatorsList(response.indicators);
+                }
+            });
+        },
+
+        /**
+         * @private @method _getIndicatorById
+         * Finds indicator object matching given id.
+         * Shows an error message if no matches found.
+         *
+         * @param {Number} id indicator id
+         * @return {Object} matching indicator object or undefined if not found
+         */
+        _getIndicatorById: function (id) {
+            var matches = this.indicatorData.filter(function (indicator) {
+                return indicator.id === id;
+            });
+            if (matches.length > 0) {
+                return matches[0];
+            }
+            // couldn't find indicator -> show an error
+            this._showErrorMessage(this.loc('tab.error.notfound'));
+        },
+
+        /**
+         * @private @method _confirmDelete
+         * Shows a confirmation dialog on deleting a indicator
+         *
+         * @param {Object} indicator data object for the indicator to delete
+         */
+        _confirmDelete: function (indicator) {
+            var me = this;
+            var dialog = Oskari.clazz.create('Oskari.userinterface.component.Popup');
+            var okBtn = Oskari.clazz.create('Oskari.userinterface.component.Button');
+
+            okBtn.setTitle(me.loc('tab.delete'));
```

Comment:
You can use Oskari.userinterface.component.buttons.DeleteButton instead so we don't need to duplicate the localization

Reply:
But I guess we need the localization outside the button as well so it's ok to have the localization here.